### PR TITLE
blugon: init at 1.11.4

### PR DESCRIPTION
**Pull request by jumper149**
###### Motivation for this change
Adding a new application.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date (there is only metadata)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/misc/blugon/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3, libX11, libXrandr, gcc, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "blugon";
+  version = "1.11.4";
+
+  src = fetchFromGitHub {
+    owner = "jumper149";
+    repo = "blugon";
+    rev = "c53b47eac7f9b23adeb62f8df10f38ffbc0dd156";
```
Comment (JohnAZoidberg) [inline pkgs/applications/misc/blugon/default.nix]: Please tag a release on GitHub and refer to it here.

**Review by JohnAZoidberg — CHANGES_REQUESTED** [review]
Thanks for your contribution! :)

Tested and it seems to work.
Is the filter supposed to stay after I kill the program?
Got my natural color back with `blugon --setcurrent=6600.0`

**Comment by jumper149** [comment]
> Is the filter supposed to stay after I kill the program?
> Got my natural color back with `blugon --setcurrent=6600.0`

Yes this just the way it's intended to be!

File: pkgs/applications/misc/blugon/default.nix (hunk 1 above)
Comment (jumper149) [reply pkgs/applications/misc/blugon/default.nix]: Created a [release](https://github.com/jumper149/blugon/tree/1.11.4).
Commit hash stays the same.

File: pkgs/applications/misc/blugon/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    sha256 = "0x320w2h5nlcgha4345i8ns15akb4kmrdgkh710s4r1n1by4x11r";
+  };
+
+  buildInputs = [ python3 libX11 libXrandr gcc bash ];
```
[the remark this answers is not in the document]
Comment (jumper149) [reply pkgs/applications/misc/blugon/default.nix]: Done.

**Comment by jumper149** [comment]
I used `git rebase` to fixup the previous commit.

File: pkgs/applications/misc/blugon/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3, libX11, libXrandr }:
+
+stdenv.mkDerivation rec {
+  pname = "blugon";
+  version = "1.11.4";
+
+  src = fetchFromGitHub {
+    owner = "jumper149";
+    repo = "blugon";
+    rev = "c53b47eac7f9b23adeb62f8df10f38ffbc0dd156";
```
Comment (JohnAZoidberg) [inline pkgs/applications/misc/blugon/default.nix]:
```suggestion
    rev = version;
```

File: pkgs/applications/misc/blugon/default.nix [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  meta = with stdenv.lib; {
+    description = "Simple and configurable Blue Light Filter for X";
+    longDescription = ''
+      blugon is a simple and fast Blue Light Filter, that is highly configurable and provides a command line interface.
+      The program can be run just once or as a daemon (manually or via systemd).
+      There are several different backends available.
+      blugon calculates the screen color from your local time and configuration.
+    '';
+    license = licenses.asl20;
+    homepage = https://github.com/jumper149/blugon;
```
Comment (JohnAZoidberg) [inline pkgs/applications/misc/blugon/default.nix]: Looks like [RFC45](https://github.com/NixOS/rfcs/pull/45) is finally getting approved.
```suggestion
    homepage = "https://github.com/jumper149/blugon";
```

File: pkgs/applications/misc/blugon/default.nix [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  src = fetchFromGitHub {
+    owner = "jumper149";
+    repo = "blugon";
+    rev = "c53b47eac7f9b23adeb62f8df10f38ffbc0dd156";
+    sha256 = "0x320w2h5nlcgha4345i8ns15akb4kmrdgkh710s4r1n1by4x11r";
+  };
+
+  buildInputs = [ python3 libX11 libXrandr ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
```
Comment (JohnAZoidberg) [inline pkgs/applications/misc/blugon/default.nix]:
```suggestion
  installFlags = [ "PREFIX=$(out)" ];
```

File: pkgs/applications/misc/blugon/default.nix [hunk 6]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3, libX11, libXrandr }:
+
+stdenv.mkDerivation rec {
+  pname = "blugon";
+  version = "1.11.4";
+
+  src = fetchFromGitHub {
+    owner = "jumper149";
+    repo = "blugon";
```
Comment (JohnAZoidberg) [inline pkgs/applications/misc/blugon/default.nix]:
```suggestion
    repo = pname;
```

**Review by JohnAZoidberg — CHANGES_REQUESTED** [review]
Some minor things, then it should be okay :)

File: pkgs/applications/misc/blugon/default.nix (hunk 5 above)
Comment (jumper149) [reply pkgs/applications/misc/blugon/default.nix]: The `PREFIX` is also required for `make build`.

File: pkgs/applications/misc/blugon/default.nix (hunk 3 above)
Comment (jumper149) [reply pkgs/applications/misc/blugon/default.nix]: Ok, I should've checked the nixpkgs manual when you said I should use the version tag. Done.

File: pkgs/applications/misc/blugon/default.nix (hunk 4 above)
Comment (jumper149) [reply pkgs/applications/misc/blugon/default.nix]: Done.

**Comment by ryantm** [comment]
Thanks!

**Pull request merged**